docs/source/markdown/podman-create.1.md

```diff
@@ -881,6 +886,37 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Podman to label the content with a private unshared label.
 Only the current container can use a private volume.
 
+  `Overlay Volume Mounts`
+
+   The `:O` flag tells Podman to mount the directory from the host as a
+temporary storage using the Overlay file system. The containers
+are allowed to modify contents within the mountpoint and are stored in the
+container storage in a separate directory.  In Overlay FS terms the source
+directory will be the lower, and the container storage directory will be the
+upper. Modifications to the mount point are destroyed when the container
+finish executing, similar to a tmpfs mount point being unmounted.
+
+  Subsequent execution of the container will see the original source directory
+content, any changes from previous container executions no longer exists.
+
+  One use case of the `overlay` mount is sharing the package cache from the
+host into the container to allow speeding up builds.
+
+  Note:
+
+     - The `O` flag is not allowed to be specified with the `Z` or `z` flags.
+Content mounted into the container is labeled with the private label.
+       On SELinux systems, labels in the source directory needs to be readable
+by the container label. Usually containers can read/execute `container_share_t`
+and can read/write `container_file_t`. If you can not change the labels on
+source volume, SELinux container separation must be disabled for the container
+to work.
+     - Modification of the directory volume mounted into the container with an
```

Comment:
This is a bit hard to brain parse. Am I reading it correctly that we should not modify the mounted directory on the _host_?

Reply:
Changed to:
```
     - The source directory mounted into the container with an overlay mount
should not be modified, it can cause unexpected failures.  It is recommended
that you do not modify the directory until the container finishes running.
```